tasks/43_brackets-indexes/brackets-indexes.js

```diff
@@ -0,0 +1,21 @@
+export const findCloseIndex = (str, pos) => {
+  let openBracketCount = 0;
+  let givenBracketPosition = -1;
+  let index = -1;
+  for (let i = 0; i < str.length; i++) {
```

Comment:
Are we sure this is the most efficient and readable solution? Hint: do we have to start from index 0?